Introduction of the WIA/KA revocation (#113)
APTITUDE-Consortium/wp2-trust-specifications: docs/topics/revocation-mechanisms.md

    - make available an endpoint to request [Status List Tokens](#status-list-token);
    - issue <artifacts:Wallet-Relying Party Registration Certificate (WRPRC)|WRPRCs> with the appropriate parameter `status` as described in [Wallet-Relying Party Registration Certificate](../sections/trust-artifacts.md#wallet-relying-party-registration-certificate).
- To manage <artifacts:Wallet Unit Attestation (WUA)|Wallet Unit Attestations (WUAs)>, each <roles:Wallet Provider (WP)|Wallet Provider> SHALL:
    - make available an endpoint to request [Status List Tokens](#status-list-token);
    - issue <artifacts:Wallet Instance Attestation (WIA)|WIAs> and <artifacts:Key Attestation (KA)|KAs> with the appropriate status-list references in `client_status.status.status_list` and `key_storage_status.status.status_list`, respectively, as described in [Token Status List (Wallet Unit Attestation Profile)](#token-status-list-wallet-unit-attestation-profile).

### Token Status List

> This section profiles the Token Status List (TSL) mechanism of [draft-ietf-oauth-status-list](https://datatracker.ietf.org/doc/draft-ietf-oauth-status-list/) for artifacts whose status is managed using a Status List. A TSL conveys the current status of many artifacts in a compact, signed <artifacts:Status List Token>. The requirements in this profile apply unless refined by one of the artifact-specific profiles below.

Author: ```suggestion
This section profiles the Token Status List (TSL) mechanism of [draft-ietf-oauth-status-list](https://www.ietf.org/archive/id/draft-ietf-oauth-status-list-21.html) for artifacts whose status is managed using a Status List. A TSL conveys the current status of many artifacts in a compact, signed <artifacts:Status List Token>. The requirements in this profile apply unless refined by one of the artifact-specific profiles below.
```

This profile distinguishes between:

- **Runtime status validation**, performed as part of the credential issuance transaction and described in RFC001.
- **Operational status monitoring**, performed by the PID or Attestaion Provider after issuance and during the technical validity period of the issued credential.

!!! choice

    For a device-bound credential, both the WIA and the KA are applicable, and the statuses of both SHALL be checked at issuance.
    
    For a non-device-bound credential, requirements concerning the KA do not apply, and only the status of the WIA SHALL be checked at issuance.

For PIDs, the current [TS03] requirements, as reflected in [CIR 2026/1731](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=OJ%3AL_202601731), require the PID Provider to monitor both the WIA and KA status at least once every 24 hours when the technical validity period of the PID exceeds 24 hours.

!!! choice

    For other Attestation types, within the APTITUDE profiles,
    

>     - both WIA and KA operational checks SHALL be performed for device-bound credentials when operational monitoring is enabled;

Author: ```suggestion
    - both WIA and KA operational checks SHALL be performed for device-bound credentials when operational monitoring is enabled; while,
```

>     - while only the WIA operational check SHALL be performed for non-device-bound attestations when operational monitoring is enabled.

Author: ```suggestion
    - only the WIA operational check SHALL be performed for non-device-bound attestations when operational monitoring is enabled.
```
